docs/01_nodeos/03_plugins/net_plugin/index.md

```diff
@@ -23,77 +23,77 @@ Config Options for eosio::net_plugin:
   --p2p-listen-endpoint arg (=0.0.0.0:9876)
                                         The actual host:port used to listen for
                                         incoming p2p connections.
-  --p2p-server-address arg              An externally accessible host:port for 
-                                        identifying this node. Defaults to 
+  --p2p-server-address arg              An externally accessible host:port for
+                                        identifying this node. Defaults to
                                         p2p-listen-endpoint.
-  --p2p-peer-address arg                The public endpoint of a peer node to 
-                                        connect to. Use multiple 
-                                        p2p-peer-address options as needed to 
+  --p2p-peer-address arg                The public endpoint of a peer node to
+                                        connect to. Use multiple
+                                        p2p-peer-address options as needed to
                                         compose a network.
                                           Syntax: host:port[:<trx>|<blk>]
-                                          The optional 'trx' and 'blk' 
-                                        indicates to node that only 
-                                        transactions 'trx' or blocks 'blk' 
+                                          The optional 'trx' and 'blk'
+                                        indicates to node that only
+                                        transactions 'trx' or blocks 'blk'
                                         should be sent.  Examples:
-                                            p2p.eos.io:9876
-                                            p2p.trx.eos.io:9876:trx
-                                            p2p.blk.eos.io:9876:blk
-                                        
+                                            p2p.domain.io:9876
+                                            p2p.trx.domaine.io:9876:trx
```

Comment:
misspelling of 'domain'

Reply:
Use a proper example domain as per RFC2606

https://www.rfc-editor.org/rfc/rfc2606.html